packages/stylehacks/src/plugins/bodyEmpty.js

```diff
@@ -1,33 +1,41 @@
 import parser from 'postcss-selector-parser';
 import exists from '../exists';
 import isMixin from '../isMixin';
-import plugin from '../plugin';
+import Plugin from '../plugin';
 import { FF_2 } from '../dictionary/browsers';
 import { SELECTOR } from '../dictionary/identifiers';
 import { RULE } from '../dictionary/postcss';
 import { BODY } from '../dictionary/tags';
 
-function analyse(ctx, rule) {
```

Comment:
This is going to receive `this` as the `ctx` argument when called inside the class, so it is the same as an instance method.